templates/terraform/cloudsql/main.tf

```diff
@@ -49,7 +49,7 @@ resource google_sql_user db_user {
   for_each = toset(var.user_names)
 
   provider = google.target
-  name = "${var.name_prefix}-${each.value}"
+  name = each.value
```

Comment:
Any reason in particular why we're dropping the name prefix for these?

Reply:
I never intended for the name prefixes to exist for these (missed it in previous review).